lib/GoogleTest/GoogleTestFinder.cpp

```diff
@@ -98,12 +95,12 @@ std::vector<std::unique_ptr<Test>> GoogleTestFinder::findTests(Context &Ctx) {
         continue;
       }
 
-      Type *SeqTy = Ty->getSequentialElementType();
```

Comment:
This also looks like a rebase artifact. The master branch should now have `getPointerElementType` (https://github.com/mull-project/mull/blob/master/lib/GoogleTest/GoogleTestFinder.cpp#L101).